fic_modules/git.py

```diff
@@ -428,10 +440,12 @@ def get_date_from_json(repo_name):
             .get("latest_release") \
             .get("date")
         date_format = parse(last_stored_date)
-        last_stored_date = datetime.strptime(str(date_format), "%Y-%m-%d %H:%M:%S")
+        last_stored_date = datetime \
+            .strptime(str(date_format), "%Y-%m-%d %H:%M:%S")
     except FileNotFoundError:
-        last_stored_date = datetime.strptime("2019-01-01 01:00:00", "%Y-%m-%d %H:%M:%S")
-    logger.info("last local date was: {} ".format(last_stored_date))
+        last_stored_date = datetime \
+            .strptime("2019-01-01 01:00:00", "%Y-%m-%d %H:%M:%S")
+        LOGGER.info("last local date was: {} ".format(last_stored_date))
```

Comment:
Compared to old revision, this gained Indentation. 
Was it on porpoise? (maybe a bug was fixed, which is why I'm asking)

Reply:
That looks to be a display issue here on github.
Just got the raw version locally (from this commit) and it doesn't contain any extra indentations.
Please check the file and search for the function definition `def get_date_from_json(repo_name):`, https://raw.githubusercontent.com/mozilla-releng/firefox-infra-changelog/00bebdbaec32d3941afe21c3973d973d94d9e925/fic_modules/git.py